ClientKit/Display.cs

```diff
@@ -0,0 +1,417 @@
+                "[OSVR] DisplayConfig.GetProjectionForViewerEyeSurface(): native osvrClientGetProjectionForViewerEyeSurface call failed.");
+            return ret;
+        }
+
+        /// <summary>
+        /// Get the projection matrix for a surface seen by an eye of a viewer
+        /// in a display config. (float version)
+        /// </summary>
+        /// <param name="viewer">Viewer ID</param>
+        /// <param name="eye">Eye ID</param>
+        /// <param name="surface">Surface ID</param>
+        /// <param name="near">Distance to near clipping plane - must be nonzero, typically positive.</param>
+        /// <param name="far">Distance to far clipping plane - must be nonzero, typically
+        ///     positive and greater than near.</param>
+        /// <param name="flags">Bitwise OR of matrix convention flags (see OSVR_MatrixFlags)</param>
+        public Matrix44f GetProjectionMatrixForViewerEyeSurfacef(ViewerCount viewer, EyeCount eye, SurfaceCount surface, float near, float far, MatrixConventionsFlags flags)
+        {
+            Matrix44f ret;
+            CheckSuccess(
+                DisplayConfigNative.osvrClientGetViewerEyeSurfaceProjectionMatrixf(mHandle, viewer, eye, surface, near, far, flags, out ret),
+                "[OSVR] DisplayConfig.GetProjectionForViewerEyeSurface(): native osvrClientGetProjectionForViewerEyeSurface call failed.");
+            return ret;
+        }
+
+        /// <summary>
+        /// Determines if a surface seen by an eye of a viewer in a display
+        /// config requests some distortion to be performed.
+        /// 
+        /// This simply reports true or false, and does not specify which kind of
+        /// distortion implementations have been parameterized for this display. For
+        /// each distortion implementation your application supports, you'll want to
+        /// call the function to retrieve the parameters to find out if it is available.
+        /// </summary>
+        /// <param name="viewer">Viewer ID</param>
+        /// <param name="eye">Eye ID</param>
+        /// <param name="surface">Surface ID</param>
+        public bool DoesViewerEyeSurfaceWantDistortion(ViewerCount viewer, EyeCount eye, SurfaceCount surface)
+        {
+            bool ret;
+            CheckSuccess(
+                DisplayConfigNative.osvrClientDoesViewerEyeSurfaceWantDistortion(mHandle, viewer, eye, surface, out ret),
+                "[OSVR] DisplayConfig.DoesViewerEyeSurfaceWantDistortion(): native osvrClientDoesViewerEyeSurfaceWantDistortion call failed.");
+            return ret;
+        }
+
+        /// <summary>
+        /// Returns the radial distortion parameters, if known/requested, for a
+        /// surface seen by an eye of a viewer in a display config.
+        /// 
+        /// NOTE: This method will throw an exception if the surface does not have

```

Comment:
This is probably not the right way to handle this, since it's not an exceptional case. Do we want a separate method (in c api or in the wrapper) to check if radial distortion params are supported?